zellij-utils/src/snapshots/zellij_utils__setup__setup_test__default_config_with_no_cli_arguments.snap

```diff
@@ -974,7 +974,79 @@ Config {
                 'n',
             ): [
                 NewTab(
-                    None,
+                    Some(
```

Comment:
I'm not thrilled about this... :)

Can we maybe get around this by always returning `NewTab(None)` from the KDL parser unless there's a layout, cwd or name specified explicitly?

Reply:
Yes. It's possible. And I understand your concerns too. Thanks!